cli/utils.js

```diff
@@ -46,7 +46,7 @@ const error = function(msg, trace) {
     if (trace) {
         console.log(trace);
     }
-    process.exit(1);
+    return; // eslint-disable-line
```

Comment:
@jamesgeorge007  I don't get the point of this, how is the process killed on error now? this was to ensure we don't have any handing process/subprocess

Reply:
@deepu105 It was specific to the case here. I guess we could replace `logger.error()` with `logger.info()` :thinking:

Reply:
What would you suggest?

Reply:
Keep the process exit and do console log directly for the usage in question